CITATION.cff

```diff
@@ -0,0 +1,31 @@
+cff-version: 1.2.0
+message: "Please cite the following works when using this software."
+authors:
+- family-names: "Feickert"
+  given-names: "Matthew"
+  orcid: "https://orcid.org/0000-0003-4124-7862"
+  affiliation: "University of Wisconsin-Madison"
+- family-names: "Katz"
+  given-names: "Daniel"
+  orcid: "https://orcid.org/0000-0001-5934-7525"
+  affiliation: "National Center for Supercomputing Applications"
```

Comment:
@danielskatz Do you want your affiliation as NCSA like it is now or should it be University of Illinois at Urbana-Champaign?